AGENTS.md

```
@@ -0,0 +1,62 @@
+
+- **Terraform**: After merging, apply `terraform/api_gateway.tf` to create/update API Gateway resources:
+  ```bash
+  terraform -chdir=terraform init
+  terraform -chdir=terraform plan
+  terraform -chdir=terraform apply
+  ```
+- **Lambda**: Push to `master` triggers the `deploy-lambda.yml` GitHub Action which builds and deploys the Lambda function.
+
+## Trade Lifecycle
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)